example/my-app/src/components/tables/PaginatedTable/filters/BindableValue/index.tsx

```
@@ -0,0 +1,48 @@
+    return <div className="text-sm text-gray-500">No value</div>
+  }
+
+  const bindableType = bindable.$type
+  if (!bindableType) {
+    console.error('BindableValue: bindable.$type is undefined', bindable)
+    return <div className="text-sm text-red-500">Unknown bindable type</div>
+  }
+
+  switch (bindableType) {
+    case 'predicate.BindValue':
+      const bindValue = bindable['predicate.BindValue']
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)